src/components/tooltip/Tooltip.tsx

```diff
@@ -110,6 +112,7 @@ class Tooltip extends React.Component<TooltipProps, State> {
         constrainToScrollParent: false,
         constrainToWindow: true,
         isDisabled: false,
+        offset: '0 0',
```

Comment:
https://github.com/box/box-ui-elements/blob/master/src/components/dropdown-menu/DropdownMenu.js has an example that useBubble was defined in a way that it will not change snapshot and avoid being a breaking change.

Reply:
updated to make optional to not break snapshots